pkg/apis/management.cattle.io/v3/global_types.go

```diff
@@ -21,6 +21,7 @@ type Setting struct {
 
 // +genclient
 // +genclient:nonNamespaced
+// +kubebuilder:resource:scope=Cluster
```

Comment:
why did you need to add this? Would this break or change existing functionality?